src/ubuntu/14.04/crossdeps/hooks/post-build

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$(dirname ${BASH_SOURCE[0]})/../../../build-scripts/build-install-binfmt-support.sh ubuntu-16.04
```

Comment:
Last arg looks incorrect - shouldn't it be 14.04

Reply:
Acknowledged